docs/rst/manual/about/release_notes.rst

```diff
@@ -27,6 +27,36 @@ Release Notes
  Miscellaneous:
    -
 
+Version 2.27
+------------
+
+Bugfixes:
+  - Include last report step in HISTORY_OBSERVATION (`#1820 <https://github.com/equinor/ert/issues/1820>`_)
```

Comment:
@oyvindeide says this is a breaking change, so should be labelled as such.

Reply:
Yes, this change does impact the results from ERT. But, it is also a bugfix. Do we want to communicate all bugfixes that changes behaviour as breaking changes?

Reply:
@jondequinor & @oyvindeide: I still consider the change to be a bugfix, but I've added a comment also under breaking changes pointing to the bugfix and explaining that it is expected to cause changes in numerical results. Hopefully addresses your concerns. If not, feel free to patch it 👍🏻